information/gmt_data_server.txt

```diff
@@ -0,0 +1,59 @@
+/server/earth_relief	earth_relief_02m_g.grd	02m	g	 71M	Earth Relief at 2x2 arc minutes from Gaussian Cartesian filtering (3.7 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_01m_g.grd	01m	g	258M	Earth Relief at 1x1 arc minutes from Gaussian Cartesian filtering (1.9 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_30s_g.grd	30s	g	935M	Earth Relief at 30x30 arc seconds from Gaussian Cartesian filtering (1.0 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_60m_p.grd	60m	p	128K	Earth Relief at 60x60 arc minutes from Gaussian Cartesian filtering (111 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_01d_p.grd	01d	p	128K	Earth Relief at 1x1 arc degrees from Gaussian Cartesian filtering (111 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_30m_p.grd	30m	p	435K	Earth Relief at 30x30 arc minutes from Gaussian Cartesian filtering (55 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_20m_p.grd	20m	p	918K	Earth Relief at 20x20 arc minutes from Gaussian Cartesian filtering (37 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_15m_p.grd	15m	p	1.6M	Earth Relief at 15x15 arc minutes from Gaussian Cartesian filtering (28 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_10m_p.grd	10m	p	3.4M	Earth Relief at 10x10 arc minutes from Gaussian Cartesian filtering (18 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_06m_p.grd	06m	p	8.8M	Earth Relief at 6x6 arc minutes from Gaussian Cartesian filtering (10 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_05m_p.grd	05m	p	 13M	Earth Relief at 5x5 arc minutes from Gaussian Cartesian filtering (9 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_04m_p.grd	04m	p	 19M	Earth Relief at 4x4 arc minutes from Gaussian Cartesian filtering (7.5 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_03m_p.grd	03m	p	 33M	Earth Relief at 3x3 arc minutes from Gaussian Cartesian filtering (5.6 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_02m_p.grd	02m	p	 71M	Earth Relief at 2x2 arc minutes from Gaussian Cartesian filtering (3.7 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_01m_p.grd	01m	p	258M	Earth Relief at 1x1 arc minutes from Gaussian Cartesian filtering (1.9 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_30s_p.grd	30s	p	935M	Earth Relief at 30x30 arc seconds from Gaussian Cartesian filtering (1.0 km fullwidth) of SRTM15+V2.1 [Tozer et al., 2019]
+/server/earth_relief	earth_relief_15s_p.grd	15s	p	3.2G	Earth Relief at 15x15 arc seconds provided by SRTM15+V2.1 [Tozer et al., 2019]
```

Comment:
Again, no gridline registration for 15s. Is it on purpose?

Reply:
Yes.  The 15s pixel is the source.  We cannot derive a gridline file form that without loosing all the information that sets it apart from the 30s file.  The only real solution is to ask Dave and Brook to also create a 15s gridline file from their original sources.